membership-attribute-service/app/prodtest/Allocator.scala

```diff
@@ -0,0 +1,12 @@
+package prodtest
+
+object Allocator {
+
+  def isInTest(identityId: String, percentageInTest: Double): Boolean = {
+    val cleaned = identityId.replaceFirst("^0+", "").toInt
```

Comment:
can identityId ever be an invalid int? and if so will we end up with the correct response?